Tests/Functional/Frontend/DemonstrationPageTest.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+declare(strict_types=1);
+
+namespace B13\Picture\Tests\Functional\Frontend;
+
+/*
+ * This file is part of TYPO3 CMS-based extension "doktypemapper" by b13.
+ *
+ * It is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License, either version 2
+ * of the License, or any later version.
+ */
+
+use TYPO3\TestingFramework\Core\Functional\Framework\Frontend\InternalRequest;
+use TYPO3\TestingFramework\Core\Functional\FunctionalTestCase;
+
+class ImageViewHelperTest extends FunctionalTestCase
+{
+    protected $pathsToLinkInTestInstance = [
+        'typo3conf/ext/picture/Build/sites' => 'typo3conf/sites',
+    ];
+
+    protected $testExtensionsToLoad = [
+        'typo3conf/ext/picture',
+    ];
+
+    /**
+     * @test
+     */
+    public function callingPageReturns200ResponseCode(): void
+    {
+        $this->importDataSet(ORIGINAL_ROOT . 'typo3conf/ext/picture/Tests/Functional/Frontend/Fixtures/demonstration_page.xml');
```

Comment:
tipp: switch to importCSVDataSet() since importDataSet() will vanish in v12 compat TF